dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java

```diff
@@ -230,6 +236,32 @@ public static Tool convertEntryToTool(Entry<?, ?> container, DockstoreWebservice
         return tool;
     }
 
+    private static List<ImageData> processImageDataForWorkflowVersions(final Version version, final ToolVersion toolVersion) {
+        toolVersion.setIsProduction(true);
```

Comment:
I would get rid of this (see above comment). This is a side-effect and generally not a recommended practice -- I would expect a method with a signature like this to return a List<ImageData>, not return a List and modify an object.